modules/installation-aws-config-yaml.adoc

```diff
@@ -82,3 +82,8 @@ provides the default value.
 machines in your cluster.
 <4> To configure faster storage for etcd, especially for larger clusters, set the
 storage type as `io1` and set `iops` to `2000`.
+<5> For a CIDR in the range of 10.128.0.0/14, IPs in this range are, by default,
+ assigned to the Pods. If you want to change this, you must adjust the
+ `ClusterNetwork`.
+ <6> To customize the IPs rolled out to services, then you must adjust the
```

Comment:
s/IPs/IP addresses

Can we have another sentence or two of context here?

Reply:
A block of IP addresses for services. OpenShiftSDN allows only one serviceNetwork block. The address block must not overlap with any other network block. 
Given that, this is all a part of install procedure currently and where much documented, we do not want to duplicate these steps.